source/patterns/@aws-solutions-constructs/core/lib/dynamodb-table-helper.ts

```diff
@@ -49,15 +56,19 @@ export interface BuildDynamoDBTableWithStreamProps {
   readonly existingTableInterface?: dynamodb.ITable
 }
 
-export function buildDynamoDBTable(scope: Construct, props: BuildDynamoDBTableProps): dynamodb.Table {
+export function buildDynamoDBTable(scope: Construct, props: BuildDynamoDBTableProps): [dynamodb.ITable, dynamodb.Table?] {
   // Conditional DynamoDB Table creation
```

Comment:
let's add a check here that they didn't provide a table AND and interface and just throw an error if they did.